.ci/magic-modules/helpers.sh

```diff
@@ -16,5 +16,5 @@ function apply_patches {
   # We keep all the changes (--soft).
   git reset --soft "$(git merge-base HEAD "$4")"
   # Then we commit again.
-  git commit -m "$2" --author="$3" || true  # don't crash if no changes
+  git commit -m "$2" --author="$3" --signoff || true  # don't crash if no changes
```

Comment:
Ideally you should make sure that the signoff line doesn't appear in downstream PR messages, as well.

Reply:
e.g. https://github.com/modular-magician/inspec-gcp/pull/54#issue-242386371